README.md

```diff
@@ -110,11 +104,15 @@ git@github.com:ita-social-projects/StreetCode.git
 12. Change password to the default system one - **"Admin@1234"**. Don't forget to confirm it afterwards
 13. On the left-hand side select **"Status"** page, and set **"Login"** radio-button to **"Enabled"**
 14. Click "Ok"
+15. Right click on **"localhost"** server on the left-hand side of the UI and click **"Restart"**
 
 Now you can connect to your localhost instance with login (sa) and password (Admin@1234)!
 
-**_NOTE:_** Here's the full walkthrough: https://www.youtube.com/watch?v=ANFnDqe4JBk&t=211s.
+### Troubleshooting
 
+- If you encounter an unhandled `Microsoft.Data.SqlClient.SqlException: 'A connection was successfully established with the server, but then an error occurred during the login process. (provider: SSL Provider, error: 0 - The certificate chain was issued by an authority that is not trusted.)'` while seeding the DB check if you have `TrustServerCertificate=true;` parameter in your connection string and add if you haven't.
```

Comment:
Consider rephrasing the sentence for clarity: "...while seeding the DB, check if you have the 'TrustServerCertificate=true;' parameter in your connection string and add it if you don't"